AnyAnimation/ManualAnimator.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManualAnimator.swift
+//  AnyAnimation
+//
+//  Created by Sergey Gavrilyuk on 2017-09-05.
+//  Copyright © 2017 Gavrix. All rights reserved.
+//
+
+import Foundation
+
+public class ManualAnimator: Animator {
+  
+  struct EmptyAnimation: Animation {
+    var duration: TimeInterval = 0
+    func tick(at time: RelativeTimeInterval) {}
+  }
+  
+  var currentAnimation: Animation = EmptyAnimation()
```

Comment:
Just curious why `EmptyAnimation` instead of `nil`?

Reply:
By design, it is `currentAnimation` is always supposed to be there, after `run` has been run at least once. So I decided to carry `EmptyAnimation` to avoid optionality.
No I think that having redundant type may be not worth it 🤔